### DISCUSSION TITLE
'pip install molecule-plugins[docker]' does not work as expected

### DISCUSSION THREAD

**Discussion by JustinSenia342** [ansible-community/molecule-plugins#135] [docker]
The molecule docker driver worked just fine for me yesterday, but today it's displaying some unexpected behavior that resulted in the "molecule test" command kicking back an error.

I compared the logs between yesterday and today, and the found a few things which stuck out in the logs:

At the start of the run I now get a:
**WARNING  docker driver is not installed**
_(which wasn't there yesterday - when "molecule test" still functioned as expected)_

At the end after the stack trace:
**AttributeError: module 'molecule.command' has no attribute 'lint'. Did you mean: 'list'?**

I'm guessing it's kicking back a missing attribute message is due to an issue with the docker driver install process (can't post the full stack trace though because the error is from a work computer).

I made sure the dependencies were installed, and that didn't seem to be the problem, it looks like the only change may have been a recent update to this plugin - which looks like it may not install the docker driver correctly now for some reason.

If you could look into this, it would be very much appreciated :)

**Comment by apatard** [comment]
I doubt I'll have time to look at that today, but at least, what I can say:
- 'lint' configuration option of molecule is gone (and the molecule lint command). See the release changelog.
- I've seen in my mails https://github.com/ansible-community/molecule/issues/3883 which may be similar to the other warning you're having. Don't know. I'm not sure why it's happening and I dont see any message about schema, otherwise I would have thought about a side effect of https://github.com/ansible-community/molecule/pull/3765.

**Comment by carlosmmatos** [comment]
I ran into this today and fixed it by removing any lint configurations I had in my molecule.yml file. As @apatard mentioned, molecule lint is gone from v5. 

@JustinSenia342 this is an idea of what I did to fix it:

`molecule.yml`
```diff
provisioner:
   name: ansible
 verifier:
   name: ansible
-lint: |
-  set -e
-  yamllint .
-  ansible-lint .
 scenario:
   test_sequence:
     - dependency
-    - lint
     - destroy
     - syntax
     - create
```

For GH actions workflow changes:
```diff
       - name: Install dependencies
         run: |
-          sudo apt install apt-transport-https ca-certificates curl software-properties-common
+          sudo apt install apt-transport-https ca-certificates curl software-properties-common libssl-dev
           python -m pip install --upgrade pip
-          pip install molecule[docker,lint] ansible yamllint ansible-lint
+          pip install molecule "molecule-plugins[docker]" ansible ansible-core ansible-lint
 
       - name: Build/Install the collection
         uses: nick-fields/retry@v2
```
